configuration/api.go

```diff
@@ -83,11 +83,11 @@ func (a *API) UnregisterWebhook(id string, opts *ManageWebhooksDefinitionOptions
 	}, &emptyResponse{})
 }
 
-// CreateBot allows to create bot and returns its ID.
-func (a *API) CreateBot(name, avatar string, maxChats uint, defaultPriority GroupPriority, groups []*GroupConfig, ownerClientID, timezone string, workScheduler *WorkScheduler) (string, error) {
-	var resp createBotResponse
+// CreateBot allows to create bot and returns response containing its ID and secret.
+func (a *API) CreateBot(name, avatar string, maxChats uint, defaultPriority GroupPriority, groups []*GroupConfig, ownerClientID, timezone string, workScheduler *WorkScheduler) (*CreateBotResponse, error) {
```

Comment:
We can't break a method's interface in minor version update. I suggest adding a new variant, `CreateBotWithSecret`